docusaurus.config.js

```diff
@@ -94,12 +93,18 @@ const config = {
     ],
   ],
 
-  themes: [
-    'docusaurus-theme-github-codeblock'
-  ],
+  themes: ['docusaurus-theme-github-codeblock'],
   themeConfig:
     /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
-    ({
+    {
+      announcementBar: {
+        id: 'use_fishjam_cloud',
+        content:
+          'Fishjam goes to the Cloud!<br />After 1.5 years of work we decided to do a pivot and continue the project as <a href="https://fishjam.io/">Fishjam Cloud</a> making it also closed source.<br />This documentation is no longer maintained.',
```

Comment:
**suggestion:** we could also use <strong> tags to emphasize on Fishjam Cloud, what do you think?

Reply:
Strong opinion you have. Let's make as you wish.